src/components/site/Accordion.md

```diff
@@ -0,0 +1,178 @@
+```jsx
+const collapseItems = [
+  {
+    id: 1,
+    header: "Accordion Item #1",
+    subheader: "This is a subheader",
+    text: "Text goes here"
+  },
+  {
+    id: 2,
+    header: "Accordion Item #2",
+    subheader: "",
+    text: "Text goes here"
+  },
+  {
+    id: 3,
+    header: "Accordion Item #3",
+    subheader: "",
+    text: (
```

Comment:
this property it deceiving, as the value here is html, not just text. Maybe `content` is a more appropriate name?

Reply:
changed to content